<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeuroProof Stack Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QWidget" name="planeView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuPlane">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Plane</string>
     </property>
     <addaction name="actionXY"/>
     <addaction name="actionYZ"/>
     <addaction name="actionXZ"/>
    </widget>
    <addaction name="menuPlane"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>406</width>
     <height>290</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>406</width>
     <height>290</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="controlView">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>410</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QSlider" name="planeSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>19</width>
          <height>260</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>19</width>
          <height>260</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="opacitySlider">
          <property name="minimumSize">
           <size>
            <width>261</width>
            <height>19</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>261</width>
            <height>19</height>
           </size>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="modeWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>381</width>
            <height>165</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>381</width>
            <height>165</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="trainMode">
           <widget class="QWidget" name="layoutWidget_7">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>372</width>
              <height>127</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="trainLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="trainlabel">
                 <property name="minimumSize">
                  <size>
                   <width>125</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>125</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Training Mode</string>
                 </property>
                 <property name="margin">
                  <number>15</number>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLabel" name="progressLabel">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0/0 Edges Examined</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="train2Layout">
               <item row="1" column="0">
                <widget class="QPushButton" name="nextButton">
                 <property name="maximumSize">
                  <size>
                   <width>85</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Next</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="undoButton">
                 <property name="maximumSize">
                  <size>
                   <width>85</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Undo</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="currentButton">
                 <property name="maximumSize">
                  <size>
                   <width>85</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Current</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="mergeButton">
                 <property name="maximumSize">
                  <size>
                   <width>85</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Merge</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="viewMode">
           <widget class="QWidget" name="layoutWidget_8">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>7</y>
              <width>361</width>
              <height>151</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="viewLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>15</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>x</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QTextEdit" name="textX">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_2">
                 <property name="maximumSize">
                  <size>
                   <width>15</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>y</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QTextEdit" name="textY">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLabel" name="label_3">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>plane</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QTextEdit" name="textPlane">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="searchLocation">
               <property name="maximumSize">
                <size>
                 <width>131</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Search Location</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Selected Label ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelID">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelStatus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget3">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="bodyView">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>391</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="3">
       <widget class="QPushButton" name="panSet">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QTextEdit" name="textPan">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Pan Increment</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Plane Increment</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QTextEdit" name="textPlaneIncr">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="planeSet">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="2">
       <widget class="QPushButton" name="setAnnotation">
        <property name="text">
         <string>Set Annotation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="reverseButton">
        <property name="text">
         <string>Reverse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTextEdit" name="textAnnotation"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>New Session</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Session</string>
   </property>
  </action>
  <action name="actionAddGT">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add GT</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionViewOnly">
   <property name="text">
    <string>View Only</string>
   </property>
  </action>
  <action name="actionTraining">
   <property name="text">
    <string>Training</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About DVID Viewer</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="viewBodyPanel">
   <property name="text">
    <string>Body Panel</string>
   </property>
  </action>
  <action name="viewToolPanel">
   <property name="text">
    <string>Tool Panel</string>
   </property>
  </action>
  <action name="actionSaveAsProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As Session</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionXY">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>XY</string>
   </property>
  </action>
  <action name="actionYZ">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>YZ</string>
   </property>
  </action>
  <action name="actionXZ">
   <property name="text">
    <string>XZ</string>
   </property>
  </action>
  <action name="action512">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>512</string>
   </property>
  </action>
  <action name="action768">
   <property name="text">
    <string>768</string>
   </property>
  </action>
  <action name="action1024">
   <property name="text">
    <string>1024</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
